examples/local/README.md

```diff
@@ -24,13 +30,23 @@ export CLUSTER_NAME="example-cluster"
 export COMMON_DOMAIN="internal.company.com"
 export COMMON_DOMAIN_INGRESS="company.com"
 export AZURE_LOCATION="westeurope"
+export AZURE_CLIENT_ID="XXXXX"
+export AZURE_CLIENT_SECRET="XXXXX"
+export AZURE_SUBSCRIPTION_ID="XXXXX"
+export AZURE_TENANT_ID="XXXXX"
+export AZURE_TEMPLATE_URI_VERSION="master"
 
 for f in *.tmpl.yaml; do
     sed \
         -e 's|${CLUSTER_NAME}|'"${CLUSTER_NAME}"'|g' \
         -e 's|${COMMON_DOMAIN}|'"${COMMON_DOMAIN}"'|g' \
         -e 's|${COMMON_DOMAIN_INGRESS}|'"${COMMON_DOMAIN_INGRESS}"'|g' \
         -e 's|${AZURE_LOCATION}|'"${AZURE_LOCATION}"'|g' \
+        -e 's|${AZURE_CLIENT_ID}|'"${AZURE_CLIENT_ID}"'|g' \
+        -e 's|${AZURE_CLIENT_SECRET}|'"${AZURE_CLIENT_SECRET}"'|g' \
+        -e 's|${AZURE_SUBSCRIPTION_ID}|'"${AZURE_SUBSCRIPTION_ID}"'|g' \
+        -e 's|${AZURE_TENANT_ID}|'"${AZURE_TENANT_ID}"'|g' \
+        -e 's|${AZURE_TEMPLATE_URI_VERSION}|'"${AZURE_TEMPLATE_URI_VERSION}"'|g' \
```

Comment:
There is an issue in SIG operator that we want to use helm charts instead of the raw template magic. Would this apply here as well? I would say yes.

Reply:
Yes, a helm chart would be better. I'll take a look at that.